### ISSUE TITLE
Move future telcons over to Jitsi meet?

### ISSUE THREAD

**Issue by flackr** [openui/open-ui#633]
In today's telcon (2022-11-03) some technical issues lead to us needing to find an alternative platform to meet on as well as a discussion of how zoom (and discord) are not as inclusive of platforms as they could be. I suggested we use [Jitsi meet](https://meet.jit.si/) which is free, open source ([github link](https://github.com/jitsi/jitsi-meet)) and web based (not requiring / recommending specific client software).

Should we switch to this platform for future telcons? There were no issues and no concerns raised during the discussion about the platform.

**Comment by gregwhitworth** [comment]
Can we expand a bit here, I don't really have a strong affinity to any of 'em but we moved from Google Hangouts to Zoom, then to some other client when that was acting up (I don't recall which one) and would love to not switch it up numerous times if Jitsi has any issues for users.

**Comment by gregwhitworth** [comment]
Alrighty, now that we've done Google Meet and Jitsi - I wanted to provide my opinion. I'd like to utilize Jitsi, for a few reasons.

1. I can easily provide the moderator link with folks that commonly lead meetings rather than just the guest account
2. I currently pay personally for Zoom to ensure that we can have the number of folks on the call that we need. However, we may have to do something else for recordings, but we do them rarely.

**Comment by flackr** [comment]
Yeah, I quite like the openness of jitsi, and it seems to be well supported.

> 2. ... However, we may have to do something else for recordings, but we do them rarely.

Does the recording feature not work?

![Screenshot 2023-01-19 at 12 48 11 PM](https://user-images.githubusercontent.com/366761/213521491-bbbb5051-bd3b-418d-879f-e7da97b9b748.png)

**Comment by mfreed7** [comment]
Since it sounds like we tentatively "resolved" to move the ongoing meetings to Jitsi, @gregwhitworth would you mind please updating the calendar invite accordingly? It has the Jitsi link in the "location" blank, but it might help to just put it in the body of the invite. And then delete the (now old) Zoom link details.

**Comment by gregwhitworth** [comment]
@mfreed7 yes will do - closing this as we did "Resolve" to have the meetings on Jitsi

**Comment by mfreed7** [comment]
Per the discussion, I'm going to re-open this.

And I completed my action item relative to Meet: there's no current feature that would allow arbitrary attendees to join without approval. (I added a big comment and +1 to the internal bug requesting this feature. We'll see.) I did confirm that anyone on the Google Calendar invite can auto-join without approval, so perhaps we could try to just add anyone that's interested to the invite?
